From: Chunyan Zhang <zhang.chunyan@linaro.org>
Date: Wed, 17 Dec 2014 13:11:35 +0800
Subject: [PATCH 88/91] ktime.h: Introduce ktime_ms_delta
Patch-mainline: v4.0-rc1
Git-commit: 41fbf3b39d5eca01527338b4d0ee15ee1ae1023c
References: bsc#922056 FATE#318604

This patch adds a reusable time difference function which returns the
difference in millisecond, as often used in some driver code, e.g.
mtd/test, media/rc, etc.

Signed-off-by: Chunyan Zhang <zhang.chunyan@linaro.org>
Acked-by: Arnd Bergmann <arnd@linaro.org>
Cc: zhang.lyra@gmail.com
Cc: davem@davemloft.net
Cc: john.stultz@linaro.org
Cc: dborkman@redhat.com
Link: http://lkml.kernel.org/r/1418793095-18780-1-git-send-email-zhang.chunyan@linaro.org
Signed-off-by: Thomas Gleixner <tglx@linutronix.de>
Acked-by: Cho, Yu-Chen <acho@suse.com>
---
 include/linux/ktime.h |    5 +++++
 1 file changed, 5 insertions(+)
--- a/include/linux/ktime.h
+++ b/include/linux/ktime.h
@@ -345,6 +345,11 @@ static inline s64 ktime_us_delta(const k
        return ktime_to_us(ktime_sub(later, earlier));
 }
 
+static inline s64 ktime_ms_delta(const ktime_t later, const ktime_t earlier)
+{
+	return ktime_to_ms(ktime_sub(later, earlier));
+}
+
 static inline ktime_t ktime_add_us(const ktime_t kt, const u64 usec)
 {
 	return ktime_add_ns(kt, usec * NSEC_PER_USEC);
